### ISSUE TITLE
Change documentation about running the game on mordern computer

### ISSUE THREAD

**Issue by Fadoli** [ForNeVeR/overtone#44]
I'm running the game on w10 just fine, no need to startup a VM.

We can also use open-source tools like [Magpie](https://github.com/Blinue/Magpie) to scale-up the game to more "modern" resolution. + Add the ability to alt-tab without crashing.

I've used FSR upscaling with fairly good results :

![upscale_tone2](https://github.com/ForNeVeR/overtone/assets/3913349/42545ccd-b228-4052-986a-aa96ea31e4b7)

**Comment by Fadoli** [comment]
Maybe it's possible the French release of the game has some differents file, since the ISO are widely differents in sizes : 
![image](https://github.com/ForNeVeR/overtone/assets/3913349/77725bff-01c7-4406-9ce3-4f121048da62) (This can be caused by audio, texts changes and maybe few other things like that ?)

**Comment by Fadoli** [comment]
One should also use the render mod 4 in the float.cfg file

**Comment by ForNeVeR** [comment]
So nice to see you here 😃 

> I'm running the game on w10 just fine, no need to startup a VM.

Were you able to get the sound to work? This is the last reason I keep the old instructions. I am able to run it on Win11 no problem, but with no sound :(

> Maybe it's possible the French release of the game has some differents file, since the ISO are widely differents in sizes

It is also possible that the ISO files are differently prepared/packed. We better compare the actual contents after mounting the images. Or maybe there are some additional materials in the French release? It'd be interesting to know.

> One should also use the render mod 4 in the float.cfg file

I believe you refer to `dear_method 4` and yeah, that one works (to render the game in windowed mode).

**Comment by Fadoli** [comment]
I'd have gladly helped you earlier if I knew you were working on it :). I've done some research as I also had the idea of doing an engine to run the game but never got the time/will to do so.

You're correct it's `draw_method 4`, using it in windowed mode + the magpie is pretty good for scaling it fullscreen and not crash on alt-tab.

I've never had any issue regarding to sounds.

Unrelated, I've not managed to setup the project it fails due to missing packages like `SkiaSharp` I don't know how to fetch them (I believe they're all nuget right ?). I'm not too familiar with f# and recent .net stuff as I've only worked with 4.5.2 myself. It'd be nice to have a section in the readme documenting the commands and environnement expected for installing it all. (I'm on VS 2022 community)

**Comment by ForNeVeR** [comment]
> I'd have gladly helped you earlier if I knew you were working on it :). I've done some research as I also had the idea of doing an engine to run the game but never got the time/will to do so.

Yeah, I know of your work in the same direction, have used some parts of your research and even credited you in the README :)

I have only splinters of time myself, but I do work on certain gamedev resurrection hobby projects, so it's likelt this one will also get some traction.

> I've never had any issue regarding to sounds.

Ugh, that's a bummer. I still haven't managed to get a working original with sounds without VM. And the game sounds create so much atmosphere, it's sad to play silently :(

> Unrelated, I've not managed to setup the project it fails due to missing packages like SkiaSharp I don't know how to fetch them (I believe they're all nuget right ?).

Yes, all the packages are NuGet, and I expect it to just work out of the box without any problems. If anything, I am an engineer and I set up and maintain the tooling in order; the CI checks if the project builds correctly weekly, even while I am inactive. I expect it to work in VS2022. Any particular error messages you are seeing?

> It'd be nice to have a section in the readme documenting the commands and environnement expected for installing it all. (I'm on VS 2022 community)

I was going to suggest you to refer to the **Prerequisites** and **Build** sections of the readme, but actually you are right. It's a good idea to have a section on setting up the IDE. I should expect people who wanna contribute to be just game fans, and not necessarily people who have hardcode knowledge of .NET.

**Comment by ForNeVeR** [comment]
Also, regarding your initial post.

> We can also use open-source tools like [Magpie](https://github.com/Blinue/Magpie) to scale-up the game to more "modern" resolution.

I am of course thinking about getting the game in better/modern resolutions (or maybe a whole separate resource pack, to optionally untie it from the original copyrighted resources), but so far this is of a secondary priority. The main focus for a long time will be to implement the game logic and UI correctly.

I am perhaps several years behind of releasing anything usable in the scope of this project.

**Comment by Fadoli** [comment]
Found my issue, I was missing a nuget package sources (maybe because my nuget config was made by using Unity and thus was not initialised like it often gets initialized.

So in my nuget config : `%appdata%\NuGet\NuGet.Config`, I was missing :

```
  <packageSources>
    <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
  </packageSources>
```

**Comment by ForNeVeR** [comment]
It's possible to override per project. Try placing such file into `overtone` directory (near the `.sln`), and it will override the default. This way you won't mess up your default Unity setup (that may be tricky for certain projects, so better not to touch 😅).

**Comment by Lewiscowles1986** [comment]
So I've been running via DosBox X, and via Wine using DxWnd; no sound via Wine, but sound via DosBox-X (the no sound is very annoying)

**Comment by ForNeVeR** [comment]
Oh, I should try that DosBox X thing.

**Comment by Lewiscowles1986** [comment]
Here's my launch command (I setup Windows 98 following a vogons tutorial)

```
command time -l /Applications/dosbox-x.app/Contents/MacOS/dosbox-x -c 'IMGMOUNT 2 ~/Desktop/WIN98/2GB.img -fs none -size 512,63,64,1023' -c 'IMGMOUNT D ~/Desktop/tone-rebellion.iso -t iso -fs iso' -c 'boot -l C' -c 'exit' -set memsize=512 -set vmemsize=32 -set cputype=pentium_iii -set fpu=true -set core=dynamic -set cycles=max
```
